README.md

```diff
@@ -24,6 +24,20 @@ Tag represents a single item of metadata of a package. It has a well-defined nam
 
 For the purposes of this library, a tag can have associated comments. These are consecutive comment lines directly above the tag definition in a spec file.
 
+### Source
+
+Source is a source file or a downstream patch defined by a `Source`/`Patch` tag or by an entry in `%sourcelist`/`%patchlist` section.
+
+Source can be local, specified by a filename, or remote, specified by a URL. Local sources should be located in and remote sources should be downloaded to a directory referred to as `sourcedir`.
```

Comment:
The wording of the second sentence is a bit weird, at first when reading through it, my first thought was "in what?" and I realized that the rest is at the end. Maybe splitting it into 2 sentences would make it better? `Local sources should be located in a directory referred to as sourcedir. Remote sources should be downloaded to this directory.`

Reply:
Ok, I'll try to do something about it.